verilog/parser/verilog.y

```diff
@@ -6446,6 +6446,8 @@ case_modifier_opt
     { $$ = move($1); }
   | TK_unique
     { $$ = move($1); }
+  | TK_unique0
+    { $$ = move($1); } 
```

Comment:
Having reviewed the LRM grammar, `case_modifier_opt` should be no different than `unique_priority_opt` on line 6707, so let's just replace the former with the latter everywhere in this file.  In the `/* empty */` case for `unique_priority_opt` please add the missing `{ $$ = nullptr; }`.